docs/docs/migration/migration-from-v7-to-v8.md

```diff
@@ -126,8 +128,59 @@ The class-validator peer dependency has been bumped to v0.14.0:
 npx @comet/upgrade v8/update-class-validator.ts
 ```
 
+### Remove passport
+
+Remove all passport-dependencies and add @nestjs/jwt
+
+```diff title=api/package.json
+{
+    "dependencies": {
+-       "@nestjs/passport": "^9.0.0",
+-       ...other passport dependencies
++       "@nestjs/jwt": "^10.2.0",
+    }
+}
+```
+
+:::note Codemod available
+
+```sh
+npx @comet/upgrade v8/remove-passport.ts
+```
+
 :::
 
+Rename the `strategy`-factories and wrap them in `...createAuthGuardProviders()`:
+
+```diff title=api/src/auth/auth.module.ts
+-   createStaticCredentialsBasicStrategy({ ... }),
+-   createAuthProxyJwtStrategy({ ... }),
+-   createStaticCredentialsBasicStrategy({ ... }),
++   ...createAuthGuardProviders(
++       createBasicAuthService({ ... }),
++       createJwtAuthService({ ... }),
++       createStaticUserAuthService({ ... }),
++   ),
+```
+
+:::note The configuration of the AuthServices have changed slightly. Consulting the code completion should help to adapt.
```

Comment:
This doesn't look right in the deploy preview. I believe some ":::" are missing.